### PR TITLE
Fix some else exprs with indent_ternary_operator==1

### PR DESCRIPTION
**Pull request by fwojcik**
There are a number of valid expressions which do not get indented correctly when
`indent_ternary_operator` is set to 1. Rather than add many more `if()` tests to the
logic in `indent_text()` for these cases, I removed the tests for the `ChunkType` of the
chunk following a `CT_COND_COLON`. I don't think the type should matter when indenting
the 'else' branch, and this change passes the unit tests as well as a full run on my codebase.

Without this fix, all of those ternary statements, except the control group, are mis-indented:
```cpp
namespace ns1 {
int getval(void);
void baz(void);
}


int foo(int in1, int in2, float in3, double in4) {
	int a, b, c, d, e, f, g, h;

	a = (in1 < 10) ? in2 :
	    ns1::getval();

	b = (in1 < 20) ? in2 :
	    int(in3 * 0.4);

	c = (in1 < 30) ? in2 :
	    1.25 * in4;

	d = (in1 < 40) ? in2 :
	    -3;

	e = (in1 < 50) ? in2 :
	    ~3 ^ in1;

	f = (in1 < 60) ? in2 :
	    !(in1 == in2);

	g = (in1 < 70) ? in2 :
	    ++in1;

	h = (in1 < 80) ? in2 :
	                 in1 * 2;

	if ((in1 < 10) ? in2 :
	    ns1::getval()) {
		ns1::baz();
	}

	if ((in1 < 20) ? in2 :
	    int(in3 * 0.4)) {
		ns1::baz();
	}

	if ((in1 < 30) ? in2 :
	    1.25 * in4) {
		ns1::baz();
	}

	if ((in1 < 40) ? in2 :
	    -3) {
		ns1::baz();
	}

	if ((in1 < 50) ? in2 :
	    ~3 ^ in1) {
		ns1::baz();
	}

	if ((in1 < 60) ? in2 :
	    !(in1 == in2)) {
		ns1::baz();
	}

	if ((in1 < 70) ? in2 :
	    ++in1) {
		ns1::baz();
	}

	if ((in1 < 80) ? in2 :
	                 in1 * 2) {
		ns1::baz();
	}

	return a+b+c+d+e+f+g+h;
}
```

### CODE REVIEW

File: tests/expected/cpp/30353-indent_ternary-1.cpp
Comment (gmaurel) [inline tests/expected/cpp/30353-indent_ternary-1.cpp:8]: Please, only one variable per type

**Review by gmaurel — CHANGES_REQUESTED** [review]

**Comment by fwojcik** [comment]
OK, turned that into a series of variable declarations.

**Review by micheleCTDEAdmin — APPROVED** [review]
Excellent work!

**Comment by micheleCTDEAdmin** [comment]
@gmaurel 
Waiting for your approval too, before merging

**Review by gmaurel — APPROVED** [review]
Thanks

**Pull request merged**
